Userland/Utilities/headless-browser.cpp

```diff
@@ -171,6 +186,18 @@ static ErrorOr<NonnullRefPtr<Core::Timer>> load_page_for_screenshot_and_exit(Cor
     return timer;
 }
 
+static ErrorOr<URL> format_url(StringView url)
+{
+    if (Core::DeprecatedFile::exists(url))
+        return URL::create_with_file_scheme(Core::DeprecatedFile::real_path_for(url));
+
+    URL formatted_url { url };
+    if (!formatted_url.is_valid())
+        formatted_url = TRY(String::formatted("http://{}", url));
+
+    return formatted_url;
+}
```

Comment:
That seems unrelated to the commit? ("headless-browser: Add a mode to dump the layout tree after load finish")

Reply:
Sure I can separate it - this was basically "copy-paste layout tree dump from ladybird" and that included this.